e2e/README.md

```diff
@@ -0,0 +1,41 @@
+### Setup your environment
+
+In order to get your environment set up to run [Playwright](https://playwright.dev) tests, you can run `./setup-environment`, or run equivalent commands for your current setup.
+
+What this script does:
+
+-   Navigate to the folder above `mattermost-plugin-starter-template`
+-   Clone `mattermost` (if it is already cloned there, please have a clean git index to avoid issues with conflicts)
+-   Install webapp dependencies - `cd mattermost/webapp && npm i`
+-   Install Playwright test dependencies - `cd ../e2e-tests/playwright && npm i`
+-   Install plugin e2e dependencies - `cd ../../../mattermost-plugin-starter-template/e2e && npm i`
+-   Build and deploy plugin - `make deploy`
+
+---
+
+### Run the tests
+
+Start Mattermost server:
+
+-   `cd <path>/mattermost/server`
+-   `make test-data`
```

Comment:
Do CI also need to run that command?

Reply:
Not needed since Playwright generate test data depending on test requirement (on a per spec basis)